cmd/auth/login.go

```diff
@@ -43,8 +43,36 @@ func newLoginCommand(persistentAuth *auth.PersistentAuth) *cobra.Command {
 
 	cmd.RunE = func(cmd *cobra.Command, args []string) error {
 		ctx := cmd.Context()
+
+		var profileName string
```

Comment:
Profile prompt was used also to store new profile with a chosen name(or update existing one)
So for example, it would me allow to override "dogfood" profile with a new host even if I have already one.

With this change if I understand it correctly, it's not possible, right?

Reply:
Users can update the host used for a profile by passing `--host` on the CLI, so this would remove the prompt for hostname if the profile exists. My assumption is that people typically do not change the hostname for a profile; 99% of the time, they are reauthing to a host that they've already configured.